bundles/SyliusReviewBundle/forms.rst

```diff
@@ -0,0 +1,57 @@
+Forms
+=====
+
+The bundle ships with a default review type, that should be inherited by your application's specific review type.
+
+Review form
+-----------
+
+The review form type can be used wherever you need, buy using form factory. Its name depends on passed ``$subject`` parameter in constructor,
+which should correspond with your reviewable entity name.
```

Comment:
Im not sure about *correspond with*. [Consider changing to *correspond to*](http://english.stackexchange.com/questions/142495/correspond-to-vs-correspond-with)